### PR TITLE
Add the ability to mirror Text.

### PR DESCRIPTION
**Pull request by TheTwix**

### CODE REVIEW

File: KicadModTree/KicadFileHandler.py [hunk 1]
```diff
@@ -226,13 +226,16 @@ def _serialize_Text(self, node):
...
+        effects = ['effects',
+                   ['font',
+                    ['size', node.size.x, node.size.y],
+                    ['thickness', node.thickness]
+                   ]
+                  ]
+
+        if node.mirror == True:
+            effects.append(['justify', 'mirror'])
+        sexpr.append(effects)  # NOQA
```
Comment (pointhi) [inline KicadModTree/KicadFileHandler.py]: NOQA should not be required here

File: KicadModTree/KicadFileHandler.py [hunk 2]
```diff
@@ -226,13 +226,16 @@ def _serialize_Text(self, node):
...
-                     ]
-                    )  # NOQA
+        effects = ['effects',
+                   ['font',
+                    ['size', node.size.x, node.size.y],
+                    ['thickness', node.thickness]
+                   ]
+                  ]
+
+        if node.mirror == True:
```
Comment (pointhi) [inline KicadModTree/KicadFileHandler.py]: just write:

```python
if node.mirror:
```

**Comment by poeschlr** [comment]
@TheTwix are planning on fixing the issues pointed out?

**Comment by TheTwix** [comment]
Sorry, i forgot that a while ago :D

**Comment by poeschlr** [comment]
@pointhi any reservations about merging this?

**Comment by evanshultz** [comment]
It appears this also adds an attribute for text justification but not the underlying code to support setting the text justification. Not that it necessarily hurts anything, just a note.

**Comment by pointhi** [comment]
@TheTwix thanks for your contribution

**Pull request merged**